src/Controller/OverviewController.php

```diff
@@ -182,6 +182,66 @@ public function add_category(Request $request): Response
     ]);
   }
 
+  /**
+   * The page for listing all categories in the system, wether they are active or not,
+   * and then providing simple means of toggling active/inactive.
+   * Will only accessible by admins
+   * 
+   * @author Daniel Boling
+   * 
+   * @Route("/category/list", name="list_category")
+   * @IsGranted("ROLE_USER")
+   */
+  public function list_category(Request $request): Response
+  {
+
+    $em = $this->getDoctrine()->getManager();
```

Comment:
You never actually use the EntityManager in this function.  I'm guessing it's left over from a previous iteration but it should be removed since it's not needed.